app/assets/javascripts/models/knockout_extensions.js.coffee

```diff
@@ -207,3 +207,19 @@ do (ko, $=jQuery) ->
     ko.bindingHandlers.text.update element, ->
       truncatedText
     return
+
+  ko.bindingHandlers.descriptionLinks =
+    init: (element, valueAccessor, allBindings, viewModel, bindingContext) ->
+      description = $($.parseHTML(bindingContext.$data.description))
+      newDescription = ''
```

Comment:
Do we need to initialize this? It looks like we avoid returning a nil value given the if/else structure.

Reply:
Just realized you're probably referring to line 214 and not 212...

What's a better way that you recommend building this string from empty? There's probably some clean and simple way that doesn't involve initializing it like this, but I'm drawing a blank here

Reply:
Yea, `:214` -- well you are creating an empty string and then `+=` it. You could just set it in each of the if/else blocks instead of concatenating.